whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -112,7 +137,104 @@ <h3 class="title">Products</h3>
 </div>
 {% endblock content %}
 {% block footer %}
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
+    <script src="http://rawgit.com/Logicify/jquery-locationpicker-plugin/master/dist/locationpicker.jquery.js"></script>
 <script>
+
+$('#map-address').hide();
+$('#map-address-fake').hide();
+
+var dbAddress = "{{ vendor_form.street.value }}, {{ vendor_form.city.value }}, {{ vendor_form.state.value }}, {{ vendor_form.zip.value }}";
+
+{% if vendor_form.instance.pk  %}
+    $('#map-address-fake').val(dbAddress);
+{% endif %}
+
+$('#address-checkbox').click(function(){
+
+    if($(this).prop("checked") == true) {
+
+       alert("Address field locked!");
```

Comment:
@tschuy As you mentioned in your comments, I am not sure if we want to get rid of these alerts, they make the entry process more interactive. What do you think?